tests/core/snapd-refresh-vs-services-reboots/task.yaml

```diff
@@ -0,0 +1,109 @@
+summary: Check that refreshing snapd in the worst case reboots if things go sideways
+
+# TODO: move this test to tests/regression/lp-1924805 ?
+
+systems: [ubuntu-core-18-*, ubuntu-core-20-*]
+
+environment:
+  # the test needs to start from 2.49.2 to reproduce the bug and demonstrate the
+  # worst case behavior still works
+  SNAPD_2_49_2_X86: https://storage.googleapis.com/spread-snapd-tests/snaps/snapd_2.49.2_11588.snap
```

Comment:
Please update these paths to the new ones:
old -> https://storage.googleapis.com/spread-snapd-tests
new -> https://storage.googleapis.com/snapd-spread-tests

Reply:
done